pkgs/development/tools/wasm-bindgen-cli/default.nix

```diff
@@ -7,28 +7,27 @@
 , stdenv
 , curl
 , Security
-, runCommand
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "wasm-bindgen-cli";
-  version = "0.2.84";
+  version = "0.2.87";
 
   src = fetchCrate {
     inherit pname version;
-    sha256 = "sha256-0rK+Yx4/Jy44Fw5VwJ3tG243ZsyOIBBehYU54XP/JGk=";
+    hash = "sha256-0u9bl+FkXEK2b54n7/l9JOCtKo+pb42GF9E1EnAUQa0=";
   };
 
-  cargoSha256 = "sha256-vcpxcRlW1OKoD64owFF6mkxSqmNrvY+y3Ckn5UwEQ50=";
+  cargoHash = "sha256-AsZBtE2qHJqQtuCt/wCAgOoxYMfvDh8IzBPAOkYSYko=";
 
   nativeBuildInputs = [ pkg-config ];
 
   buildInputs = [ openssl ] ++ lib.optionals stdenv.isDarwin [ curl Security ];
 
   nativeCheckInputs = [ nodejs ];
 
-  # other tests require it to be ran in the wasm-bindgen monorepo
-  cargoTestFlags = [ "--test=interface-types" ];
+  # all tests require it to be ran in the wasm-bindgen monorepo
+  doCheck = false;
```

Comment:
its a bit strange, there is at least 1 test that passes, but i cant figure out how to just run that one test,


in such cases, i guess disabling all tests is ok, just hope we revisit this again